packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.stories.ts

```diff
@@ -1,18 +1,35 @@
-import { html, when } from '@microsoft/fast-element';
+import { html, ref, when } from '@microsoft/fast-element';
 import type { Meta, StoryObj } from '@storybook/html';
 import {
     createUserSelectedThemeStory,
     incubatingWarning
 } from '../../utilities/tests/storybook';
-import { richTextEditorTag } from '..';
+import { RichTextEditor, richTextEditorTag } from '..';
 import { buttonTag } from '../../button';
+import { ExampleDataType } from './types';
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 interface RichTextEditorArgs {
+    data: ExampleDataType;
     footerActionButtons: boolean;
+    getMarkdown: undefined;
+    setMarkdown: undefined;
+    editorRef: RichTextEditor;
+    setMarkdownData: (args: RichTextEditorArgs) => void;
 }
 
+const plainString = 'Plain text' as const;
+
+const markdownString = 'Supported rich text formatting options:\n1. **Bold**\n2. *Italics*\n3. Numbered lists\n   1. Option 1\n   2. Option 2\n4. Bulleted lists\n   * Option 1\n   * Option 2' as const;
```

Comment:
Please use template string so that indentations/newlines are easier to understand:
```
`Supported rich text formatting options:
1. **Bold**
2. *Italics*
...`
```

Reply:
Updated to template string.